### DISCUSSION TITLE
Meeting: Celo Wallet and (d)App Council #19 [January] 💬

### DISCUSSION THREAD

**Discussion by arthurgousset** [celo-org/celo-monorepo#10079] [Wallet and (d)App Council]
# Monthly Celo Wallet and (d)App Council

##  📆 Quick links 

- When: **Wednesday, January 11 (9am PDT/6pm CEST)**
- Where: **[Google Meet](https://meet.google.com/uwj-wfse-uzm)**
- Agenda: **comments**👇
- Previous: https://github.com/celo-org/celo-monorepo/discussions/10039
- Next: https://github.com/celo-org/celo-monorepo/discussions/10120
- Recording: [Google Drive](https://drive.google.com/file/d/1b0Hq42VwO0Ce7j8xcrUxuoslHVcWtbU6/view?usp=sharing)

## 🤷‍♂️ What is this?

A monthly meeting for Celo **developers, founders,** **product** people (anyone really!) to come together and
- find consensus (coordinate),
- learn about needs, pain points, and desires
- present and collect technical feedback

## 🗒️  Agenda

- Comment with a short sentence describing what you'd like to discuss
- Don't worry, it can be small or big! 
- Every builder is welcome (and we are very easy going, I promise 😄)
- Upvote topics you're interested in

## Housekeeping

_You can safely ignore this!_

- [x] Send update calendar invite with Github link
- [x] Encourage additions to the agenda
- [x] Publicise on forum and Discord
- [x] RECORD meeting 🎥 
- [x] Share recording below and on forum
- [ ] Create next meeting (link to next meeting above)

**Comment by jiandandekuaile** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by product-matt** [comment]
node asv2 integration demo

**Comment by product-matt** [comment]
celo roadmap, KPIs etc.

**Comment by andrewgazelka** [comment]
you really mentioning me in this bro -_-

**Comment by arthurgousset** [comment]
Here is the [recording](https://drive.google.com/file/d/1b0Hq42VwO0Ce7j8xcrUxuoslHVcWtbU6/view?usp=sharing) for future reference.
